### PR TITLE
feat: delete/purge multiple tables at once

### PR DESCRIPTION
**Pull request by relbns**
Adding new features of deleting and purging all of the tables at once. 
this is a requested feature from open issue #78 

the buttons are located on the same row as the create button but aligned to the right. 
screenshot:
image

### CODE REVIEW

File: lib/backend.js [hunk 1]
```diff
@@ -287,6 +287,35 @@ exports.createServer = (dynamodb, docClient) => {
     })
   )
 
+  app.delete('/api/deleteAllTables', asyncMiddleware(async (req, res) => {
+    try {
```
Comment (rchl) [inline lib/backend.js:291]: We don't need to explicitly use the try/catch block. `asyncMiddleware` takes care of that.

File: lib/backend.js [hunk 2]
```diff
@@ -287,6 +287,35 @@ exports.createServer = (dynamodb, docClient) => {
     })
   )
 
+  app.delete('/api/deleteAllTables', asyncMiddleware(async (req, res) => {
```
Comment (rchl) [inline lib/backend.js:290]: Not an issue but this is not very consistent route.
Maybe just `/tables` for deleting all and `/tables-prune` for pruning

**Review by rchl — CHANGES_REQUESTED** [review]

File: views/tables.ejs [hunk 3]
```diff
@@ -75,6 +75,43 @@
...
+        }
+        fetch('/api/purgeAllTables', {
+          method: 'delete'
+        }).then((response) => {
+          if (!response.ok) {
+            throw new Error
+          }
+          response.text().then(text => text && window.alert(text)
+          )
+          location.reload()
```
[the remark this answers is not in the document]
Comment (relbns) [reply views/tables.ejs:91]: is this answering the problem?

response.text().then(text => {
            text && window.alert(text)
            location.reload()
          }

[the remark this answers is not in the document]
Comment (rchl) [reply views/tables.ejs:91]:
```suggestion
          response.text().then(text => {
            window.alert(text)
            location.reload()
          })
```

**Review by rchl — APPROVED** [review]

**Pull request merged**